liblangutil/SourceReferenceFormatter.cpp

```diff
@@ -163,10 +173,18 @@ void SourceReferenceFormatter::printSourceLocation(SourceReference const& _ref)
 
 void SourceReferenceFormatter::printExceptionInformation(SourceReferenceExtractor::Message const& _msg)
 {
-	// exception header line
-	errorColored() << _msg.severity;
-	if (m_withErrorIds && _msg.errorId.has_value())
-		errorColored() << " (" << _msg.errorId.value().error << ")";
+    if (_msg.errorId.has_value())
+	{
+		if (_msg.severity == "Error")
+			errorColored() << " (" << _msg.errorId.value().error << ")";
+
+		else if (_msg.severity == "Warning")
+			warningColored() << " (" << _msg.errorId.value().error << ")";
+
+		else if (_msg.severity == "Info")
+			infoColored() << " (" << _msg.errorId.value().error << ")";
+	}
```

Comment:
You're still missing `else` here. If the severity is not one of the known three, you still need to print `_msg.errorId.value().error`.

You can use either the default color or define a new one for that case.

Reply:
Fixed !👍